lib/screens/oneOnOne/create_1on1_view.dart

```diff
@@ -492,7 +492,7 @@ class _CreateOneOnOneViewState extends State<CreateOneOnOneView> {
       maxRadius: 18.0,
       foregroundImage: CachedNetworkImageProvider(!isEmployee ? selectedEmployee.avatarAttachmentUrl ?? "" : employee.manager?.avatarAttachmentUrl ??""),
       child: Text(
-        getInitials(!isEmployee ? selectedEmployee.name ?? "" : employee.manager?.name ?? "", 2),
+        getInitials(!isEmployee ? selectedEmployee.name ?? "" : employee.manager?.name ?? "", 2).toUpperCase(),
```

Comment:
@sriLatha2328 toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.